google-cloud-language/.rubocop.yml

```diff
@@ -3,11 +3,26 @@ inherit_gem:
 
 AllCops:
   Exclude:
+    - "acceptance/**/*"
     - "google-cloud-language.gemspec"
-    - "lib/google/**/*"
+    - "lib/**/*_pb.rb"
+    - "proto_docs/**/*"
     - "Rakefile"
+    - "support/**/*"
```

Comment:
This is used for doctest, and we don't have any plans to support this in generated gems afaik.